src/commands/sbom/upload.ts

```diff
@@ -93,53 +93,58 @@ export class UploadSbomCommand extends Command {
 
     const validator: Ajv = getValidator()
 
+    if (this.basePaths.length !== 1) {
```

Comment:
I think it would make sense to change `this.basePaths` to `private basePath = Option.String()`.

Basically, [`Option.Rest()`](https://mael.dev/clipanion/docs/api/option/#optionrest) is defining **_multiple_ positional arguments**, and `required: 1` means "at least one value". You could add `arity: 1` to require "exactly one value", but `this.basePaths` would remain an array.

So I suggest you use [`Option.String()`](https://mael.dev/clipanion/docs/api/option/#optionstring-positional) _without any parameters_, which defines a single positional argument.

Reply:
Thanks I did not know that. Updated the PR based on your recommendations.